python/lsst/analysis/tools/actions/plot/focalPlanePlot.py

```diff
@@ -245,10 +268,26 @@ def makePlot(
         ax.tick_params(labelsize=7)
 
         ax.set_aspect("equal")
+
+        if self.addHistogram:
+            bins = np.linspace(
+                (self.plotMin if (self.plotMin is not None) else min(data["z"].astype(float))),
```

Comment:
If they aren't already, non-finite values should be filtered from `data["z"]` (I didn't see it being done outside of perhaps `sortAllArrays` but that's not done if not `self.showStats`.

Reply:
I opted to use the `...math.nanMin()` and `...math.nanMax()` methods.